### ISSUE TITLE
Is docker version 1.13.1 will be present on Os 7.5.1 and further future versions?

### ISSUE THREAD

**Issue by Raghuveer123** [projectatomic/docker#331]
**Output of `docker version`:**

```
(paste your output here)
```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by Raghuveer123** [comment]
any update on this?are you planning to continue in developing the future versions of docker

**Comment by rhatdan** [comment]
Kubernernetes is tied to this version of Docker, we are back porting fixes required to run kubernetes and fixes.  What features of Docker do you need that are not in this version of Docker?
Have you tried podman?

**Comment by Raghuveer123** [comment]
Nope, I have n't. But I am wondering can I run the docker version 1.13.1 on the redhat OS 7.6.
When I get a chance will try podman

**Comment by Raghuveer123** [comment]
Can any one update on this?
